wordpress-mobile/WordPress-iOS
# Integrates the latest changes from WordPressShared.

Fixes #11874 

Certain styles cause method `fontForTextStyle(_:fontWeight)` to return a font that works well with the initial setup of Dynamic Type, but that fails to update properly when the Dynamic Type size is updated in real time.

We're updating to a version of WordPressShared that fixes this issue.

## Associated PRs:

WordPressAuthenticator: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/100
WordPressShared: https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/208
WordPressKit: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/161

## Testing:

1. Launch the App
2. Create a new Site
3. Once you're offered the Quick Start Guide accept the help.
4. Tap on "Upload a Site Icon"
5. Make sure the notice that reads "Tap Your Site Icon to upload a new one" has the right font size.
6. Go to you device's settings > Large Text > Pick a very different size.
7. Go back to the App and make sure the notice has updated its size to match the settings.

## Update release notes:
